features/dataset-view.feature

```diff
@@ -330,6 +330,45 @@ Feature: a user visit the dataset page
 		| pre_03AUG2015_update 								| folder containing originally submitted data files, prior to update Aug 3rd 2015.	| Directory 		| 50.00 MiB 	|                 | ftp://climb.genomics.cn/pub/10.5524/101001_102000/101001/pre_03AUG2015_update |
 		| readme.txt 										|				                                                                    | Readme 			| 337 B 		|                 | ftp://climb.genomics.cn/pub/10.5524/101001_102000/101001/readme.txt |
 
+	@ok @javascript @pr521
```

Comment:
I know this `@pr521` tag relates to this pull request #521 but if we look at these Behat tests in the future, the pr521 tag will not mean very much to us as this pull request will have been closed. I think it would be better to rename these tags to something like `@citation-box` which is more meaningful later on.